MIDAS/src/systems.cpp

```diff
@@ -158,9 +162,29 @@ DECLARE_THREAD(buzzer, RocketSystems* arg) {
  */
 DECLARE_THREAD(kalman, RocketSystems* arg) {
     example_kf.initialize();
```

Comment:
does this need to change to displacement kf init?

Reply:
good catch, switched it